lingpy/basictypes.py

```diff
@@ -16,6 +16,10 @@ def __add__(self, other):
 
         return _strings(self._type, str(self)+' '+str(_strings(self._type, other)))
 
+    def append(self, other):
+
+        return self + other
```

Comment:
That's not how `append` is supposed to work, that's `extend`! `append` is `self + [elem]` and returns `None`. I think we should be careful here, to not violate a user's expectation/intuition based on the behaviour of python `list`.

Reply:
yeah, when writing tests, I just learned that the hard way, but this showed me, why writing tests is useful. I also figured I can't do without `super` here, will present my proposal soon.

Reply:
And I decided: if one appends, say, `[1, 2]` to `ints('1 2')` it throws an error, as the type check would fail then.